.golangci.yml

```
@@ -0,0 +1,62 @@
+# GolangCI-Lint v2 configuration file
+# See https://golangci-lint.run/usage/configuration/ :contentReference[oaicite:3]{index=3}
+
+version: "2"
+
+run:
+  timeout: 5m # tempo máximo de execução do linter :contentReference[oaicite:4]{index=4}
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)